### ISSUE TITLE
wont launch scarab; .net version 7 is too new, scarab requires .net 6

### ISSUE THREAD

**Issue by arainflake** [fifty-six/Scarab#119]
### Scarab Version

1.19.0.0-1

### Hollow Knight Version

1.5.78.11833

### Steps to Reproduce

im on linux, ran a system update which updated .net to ver 7, and now scarab wont launch. ideally i could specifiy a different directory for different .net versions, or if scarab could be made to work with .net 7.

![image](https://user-images.githubusercontent.com/74027698/209876911-64d7b6e6-c998-4cf8-87f1-a69abdb10dc9.png)

### Expected Behaviour

scarab should launch its gui

### Actual Behaviour

error and wont launch

### Logs

none

**Comment by fifty-six** [comment]
What are you running scarab from/with? The releases are self-contained so it shouldn't need a system dotnet at all. If you run the *dll* using dotnet then you'll get something like that

**Comment by arainflake** [comment]
im using an arch based distro (manjaro with the i3 window manager) and i generally launch scarab from dmenu - nothing special. the scarrab version was downloaded from manjaros official package repository as well. i resorted to launching from a terminal only to see the error.

**Comment by fifty-six** [comment]
I also run Arch. What "repo" are you getting it from? Scarab isn't in the actual package repositories. It does looks like there's two `scarab` packages in the *AUR* - but note you'd want to be using `scarab-bin`. `scarab` uses the actual installed `dotnet` which won't match the project version and will get you the error you're seeing here.

**Comment by arainflake** [comment]
Much appreciated. I was using the aur scarab package, and not scarab-bin. Switching to the later seems to have worked. I could have sworn I tried both multiple times, but now that i know to use the bin version, there are no issues. It may have been the bin version uses Scarab and non bin version uses scarab, and launching from dmenu hid the fact that when the bin was installed and I tried running lowercase scarab, nothing was found. Thank you for sticking with me on this.
